packages/nimble-components/package.json

```diff
@@ -52,6 +52,7 @@
     "@microsoft/fast-web-utilities": "^5.4.1",
     "@ni/nimble-tokens": "^4.3.2",
     "@tanstack/table-core": "^8.7.0",
+    "@tanstack/virtual-core": "^3.0.0-beta.34",
```

Comment:
Does the `beta` tag have any meaning about stability / support? Do we need anything from this latest version or would we be fine on their latest "stable" version?

Reply:
All of their versions currently available are 3.0.0 alpha or beta. 'latest' was 'alpha' so I just picked the latest 'beta' version when I started the feature.